apps/judicial-system/web/src/styles.css

```diff
@@ -0,0 +1,31 @@
+@import '../../../../libs/island-ui/fonts/fonts.css';
```

Comment:
doesnt this have to be `@import '@island.is/fonts/fonts.css'`

Reply:
I believe it is, everything is building without an error, the fonts are being downloaded and everything seems to work.